cpp/include/raft/util/bitonic_sort.cuh

```diff
@@ -18,9 +18,9 @@
 
 #include <raft/util/cuda_utils.cuh>
 
-namespace raft::spatial::knn::detail::topk {
+namespace raft::util {
```

Comment:
The `util` namespace has become part of the public APIs. Can you please add usage documentation to to the `bitonic` class?